app.js

```diff
@@ -22,14 +23,29 @@ app.post('/delta', bodyParser.json(), async (req, res) => {
       updateablePredicates.push(SUBCASE_ACTIVITY_PREDICATES[predicate]);
     }
   }
+  for (const predicate in ACTIVITY_PREDICATES) {
+    if (ACTIVITY_PREDICATES.hasOwnProperty(predicate)) {
```

Comment:
if (true) ?
Is this because the config is a key value pair and we are not looping it as such?
Loop and push just the values? 
 We don't use the named key further on, we just need the value uri

Reply:
Not sure, I'm just following the previous code here.